cfme/tests/containers/test_manageiq_ansible.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+from cfme.containers.provider import ContainersProvider
+from cfme.configure.access_control import User
+from fixtures.pytest_store import store
+from cfme.web_ui import CheckboxTable
+from utils import testgen
+from utils.ansible import setup_ansible_script, run_ansible, get_yml_value
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import toolbar as tb
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.6" and provider.version > 3.2),
+
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+providers_values_to_update = {
+    'provider_api_hostname': 'something_different.qa.lab.tlv.redhat.com'
+}
+
+provider_to_delete = 'something_different.qa.lab.tlv.redhat.com'
+
+users_values_to_create = {
+    'fullname': 'Pavel Zagalsky',
```

Comment:
I suggest using a more general user...
if it needs to be managed somewhere we can ask to add a special user for the automation